### PR TITLE
Docker Desktop update redesign docs

### PR DESCRIPTION
**Pull request by mat007**
### Proposed changes

Refresh the Docker Desktop docs about the new update.
Also updates the settings screenshots and descriptions.

@usha-mandya @stephen-turner I have merely paraphrased the descriptions under the check boxes for missing settings, I think ideally we should provide an explanation for why a user may want to choose to tick or un-tick a box, possibly with links to more docs if we have any?

### CODE REVIEW

File: docker-for-mac/index.md [hunk 1]
```diff
@@ -47,8 +47,14 @@ On the **General** tab, you can configure when to start and update Docker:
 
 - **Securely store Docker logins in macOS keychain**: Docker Desktop stores your Docker login credentials in macOS keychain by default.
 
+- **Use gRPC FUSE for file sharing**: Uncheck to use the legacy osxfs file sharing instead.
```
Comment (usha-mandya) [inline docker-for-mac/index.md:50]:
```suggestion
- **Use gRPC FUSE for file sharing**: Clear this checkbox to use the legacy osxfs file sharing instead.
```

File: docker-for-mac/index.md [hunk 2]
```diff
@@ -47,8 +47,14 @@ On the **General** tab, you can configure when to start and update Docker:
...
+
 - **Send usage statistics**: Docker Desktop sends diagnostics, crash reports, and usage data. This information helps Docker improve and troubleshoot the application. Clear the check box to opt out.
 
+* **Show weekly tips**: Display a useful advice or suggestion once a week.
```
Comment (usha-mandya) [inline docker-for-mac/index.md:54]:
```suggestion
* **Show weekly tips**: Displays useful advice and suggestions about using Docker.
```

File: docker-for-mac/index.md [hunk 3]
```diff
@@ -47,8 +47,14 @@ On the **General** tab, you can configure when to start and update Docker:
...
+
+* **Open Docker Desktop dashboard at startup**: Automatically open the dashboard when starting Docker Desktop.
```
Comment (usha-mandya) [inline docker-for-mac/index.md:56]:
```suggestion
* **Open Docker Desktop dashboard at startup**: Automatically opens the dashboard when starting Docker Desktop.
```

File: docker-for-mac/install.md [hunk 4]
```diff
@@ -66,7 +66,9 @@ and select **Learn**.
 
 Starting with Docker Desktop 3.0.0, updates to Docker Desktop will be available automatically as delta updates from the previous version.
 
-When an update is available, Docker Desktop automatically downloads it to your machine and displays an icon to indicate the availability of a newer version. All you need to do now is to click **Update and restart** from the Docker menu. This installs the latest update and restarts Docker Desktop for the changes to take effect.
+When an update is available, Docker Desktop displays an icon to indicate the availability of a newer version. Whenever convenient, you can start the download of the update in the background.
```
Comment (usha-mandya) [inline docker-for-mac/install.md:69]:
```suggestion
When an update is available, Docker Desktop displays an icon to indicate the availability of a newer version. You can start the download of the update in the background whenever it is convenient for you.
```

File: docker-for-mac/install.md [hunk 5]
```diff
@@ -66,7 +66,9 @@ and select **Learn**.
...
+
+When the download finishes, all you need to do is to click **Update and restart** from the Docker menu. This installs the latest update and restarts Docker Desktop for the changes to take effect.
```
Comment (usha-mandya) [inline docker-for-mac/install.md:71]:
```suggestion
After downloading the update, all you need to do is to click **Update and restart** from the Docker menu. This installs the latest update and restarts Docker Desktop for the changes to take effect.
```

File: docker-for-windows/index.md [hunk 6]
```diff
@@ -41,14 +41,20 @@ On the **General** tab of the Settings dialog, you can configure when to start a
 
 ![Settings](images/settings-general.png){:width="750px"}
 
-* **Start Docker when you log in** - Automatically start Docker Desktop upon Windows system login.
+* **Start Docker when you log in**: Automatically start Docker Desktop upon Windows system login.
```
Comment (usha-mandya) [inline docker-for-windows/index.md:44]:
```suggestion
* **Start Docker when you log in**: Select this option to automatically start Docker Desktop when you log into your Windows machine.
```

File: docker-for-windows/index.md [hunk 7]
```diff
@@ -41,14 +41,20 @@ On the **General** tab of the Settings dialog, you can configure when to start a
...
+* **Expose daemon on tcp://localhost:2375 without TLS**: Click this option to enable legacy clients to connect to the Docker daemon. You must use this option with caution as exposing the daemon without TLS can result in remote code execution attacks.
 
-* **Send usage statistics** - By default, Docker Desktop sends diagnostics,
+* **Use the WSL 2 based engine**: WSL 2 provides better performance than the legacy Hyper-V backend. [Learn more](wsl.md).
+
+* **Send usage statistics**: By default, Docker Desktop sends diagnostics,
 crash reports, and usage data. This information helps Docker improve and
 troubleshoot the application. Clear the check box to opt out. Docker may periodically prompt you for more information.
 
+* **Show weekly tips**: Display a useful advice or suggestion once a week.
```
Comment (usha-mandya) [inline docker-for-windows/index.md:54]:
```suggestion
* **Show weekly tips**: Displays useful advice and suggestions about using Docker.
```

File: docker-for-windows/index.md [hunk 8]
```diff
@@ -41,14 +41,20 @@ On the **General** tab of the Settings dialog, you can configure when to start a
...
+
+* **Open Docker Desktop dashboard at startup**: Automatically open the dashboard when starting Docker Desktop.
```
Comment (usha-mandya) [inline docker-for-windows/index.md:56]:
```suggestion
* **Open Docker Desktop dashboard at startup**: Automatically opens the dashboard when starting Docker Desktop.
```

File: docker-for-windows/index.md [hunk 9]
```diff
@@ -41,14 +41,20 @@ On the **General** tab of the Settings dialog, you can configure when to start a
...
 ![Settings](images/settings-general.png){:width="750px"}
 
-* **Start Docker when you log in** - Automatically start Docker Desktop upon Windows system login.
+* **Start Docker when you log in**: Automatically start Docker Desktop upon Windows system login.
 
-* **Expose daemon on tcp://localhost:2375 without TLS** - Click this option to enable legacy clients to connect to the Docker daemon. You must use this option with caution as exposing the daemon without TLS can result in remote code execution attacks.
+* **Expose daemon on tcp://localhost:2375 without TLS**: Click this option to enable legacy clients to connect to the Docker daemon. You must use this option with caution as exposing the daemon without TLS can result in remote code execution attacks.
 
-* **Send usage statistics** - By default, Docker Desktop sends diagnostics,
+* **Use the WSL 2 based engine**: WSL 2 provides better performance than the legacy Hyper-V backend. [Learn more](wsl.md).
```
Comment (usha-mandya) [inline docker-for-windows/index.md:48]:
```suggestion
* **Use the WSL 2 based engine**: WSL 2 provides better performance than the legacy Hyper-V backend. For more information, see [Docker Desktop WSL 2 backend](wsl.md).
```

**Review by stephen-turner — DISMISSED** [review]

**Review by usha-mandya — DISMISSED** [review]

File: docker-for-mac/index.md [hunk 10]
```diff
@@ -41,14 +41,20 @@ menu bar and configure the runtime options described below.
 
 On the **General** tab, you can configure when to start and update Docker:
 
+- **Automatically check for updates**: By default, Docker Desktop is configured to automatically check for newer versions. [Upgrade your existing organization to a Team plan](https://www.docker.com/pricing?utm_source=docker&utm_medium=webreferral&utm_campaign=docs_driven_upgrade) to disable the automatic check for updates. If you have installed Docker Desktop as part of an organization, you may not be able to update Docker Desktop. Consult your organization administrator to update Docker Desktop to a newer version.
```
Comment (stephen-turner) [inline docker-for-mac/index.md:44]: This paragraph seems in the wrong order. I think we should say:

```suggestion
- **Automatically check for updates**: By default, Docker Desktop is configured to check for newer versions automatically. If you have installed Docker Desktop as part of an organization, you may not be able to update Docker Desktop yourself. In that case, [upgrade your existing organization to a Team plan](https://www.docker.com/pricing?utm_source=docker&utm_medium=webreferral&utm_campaign=docs_driven_upgrade) and clear this checkbox to disable the automatic check for updates.
```

(ditto on Windows, of course).

**Review by stephen-turner — CHANGES_REQUESTED** [review]

File: docker-for-mac/index.md (hunk 10 above)
Comment (mat007) [reply docker-for-mac/index.md]: Done

**Review by stephen-turner — APPROVED** [review]
Thanks, approving now.

**Pull request merged**